addons/analytic/models/analytic_mixin.py

```diff
@@ -108,6 +112,6 @@ def _sanitize_values(self, vals, decimal_precision):
 
     def _apply_analytic_distribution_domain(self, domain):
         return [
-            ('analytic_distribution_search', leaf[1], leaf[2]) if len(leaf) == 3 and leaf[0] == 'analytic_distribution' and isinstance(leaf[2], str) else leaf
+            ('analytic_distribution_search', leaf[1], leaf[2]) if len(leaf) == 3 and leaf[0] == 'analytic_distribution' and isinstance(leaf[2], (str, tuple, list)) else leaf
```

Comment:
Good opportunity to make this function readable IMO

Reply:
I can span it over multiple lines, if that's what you mean?
But it's clear enough for me in general. Also took the same approach as the 17.0 PR, so to avoid diff ...

Reply:
Splitting it on several lines would make it readable. Your choice, as I do understand that the diff would be less readable as a tradeoff